src/virtual-wan/azext_vwan/tests/latest/test_azure_vwan_vhub_commands.py

```diff
@@ -22,3 +22,22 @@ def test_azure_vwan_vhub_basic_scenario(self, resource_group):
 
         self.cmd('network vhub update -g {rg} -n {vhub} --sku Basic')
         self.cmd('network vwan update -g {rg} -n {vwan} --type Basic')
+
+    @ResourceGroupPreparer(name_prefix='cli_test_azure_vhub_connection')
+    def test_azure_vhub_connection_basic_scenario(self, resource_group):
+        self.kwargs.update({
+            'vnet': 'clitestvnet',
+            'vwan': 'clitestvwan',
+            'vhub': 'clitestvhub',
+            'connection': 'clitestvhubconnection',
+            'rg': resource_group
+        })
+
+        self.cmd('network vnet create -g {rg} -n {vnet}')
+        self.cmd('network vwan create -n {vwan} -g {rg} --type Standard')
+        self.cmd('network vhub create -g {rg} -n {vhub} --vwan {vwan}  --address-prefix 10.5.0.0/16 -l westus --sku Standard')
+        self.cmd('network vhub connection create --resource-group {rg} --vhub-name {vhub} --name {connection} --remote-vnet {vnet}', checks=[
```

Comment:
should we add `--no-wait` in the test?

Reply:
Not too much value to test --no-wait as it returns None (Usually we don't test --no-wait)

---
In reply to: [398274212](https://github.com/Azure/azure-cli-extensions/pull/1450#discussion_r398274212) [](ancestors = 398274212)

Reply:
but this is a issue about no-wait?